wgsl/index.bs

```diff
@@ -7400,6 +7406,17 @@ for the lifetime of the variable.
 
 ## Scoped Operations ## {#scoped-operations}
 
+When an invocation performs a scoped operation, it will affect one or two sets
+of invocations.
+These sets are the <dfn noexport>memory scope</dfn> and the <dfn
+noexport>execution scope</dfn>.
+The memory scope specifies the set of invocations to which the operation will be visible.
+For [[#sync-builtin-functions|synchronization built-in functions]], this also
+means that all affected memory operations program ordered before the function
+are visible to affected operations program ordered after the function.
+The execution scope specifies the set of invocations which may participate in
```

Comment:
Suggest removing "may" here?

Reply:
I added that as future proofing against a subgroups addition. I figured this would likely be missed by such a change. I figured it's ok since all the collective operations currently describe their requirement for uniform control flow.